packages/app/src/components/DBTimeChart.tsx

```diff
@@ -285,7 +285,7 @@ function DBTimeChartComponent({
     useQueriedChartConfig(previousPeriodChartConfig, {
       placeholderData: (prev: any) => prev,
       queryKey: [queryKeyPrefix, previousPeriodChartConfig, 'chunked'],
-      enabled: enabled && config.compareToPreviousPeriod,
+      enabled: !!(enabled && config.compareToPreviousPeriod),
```

Comment:
would love a quick test added if possible. Okay to punt it if it takes longer than 5m

Reply:
Good idea, added!